lib/modules/dosomething/dosomething_reportback/dosomething_reportback.admin.inc

```diff
@@ -282,9 +284,31 @@ function dosomething_reportback_files_form_submit($form, &$form_state) {
       // Flush the image style cache for this image so they update with the new image.
       image_path_flush($image->source);
     }
+
+    // Check to see if Rogue feature flag is enabled.
+    if (module_exists('dosomething_rogue')) {
+      $rogue_item_id = db_query("SELECT rogue_rbs.rogue_item_id FROM {dosomething_rogue_reportbacks} rogue_rbs WHERE fid = :fid", array(':fid' => $fid))->fetchAll();
+
+      // If there is a rogue_item_id in the dosomething_rogue_reportbacks table, send update to Rogue.
+      if (!is_null($rogue_item_id[0]->rogue_item_id)) {
+        $data = [
+          'rogue_reportback_item_id' => $rogue_item_id[0]->rogue_item_id,
+          'status' => $values['status'],
+        ];
+
+        array_push($updates_to_send_to_rogue, $data);
+      }
+    }
   }
 
-  drupal_set_message(t("Updated."));
+  // Send updates to Rogue.
+  $response = dosomething_rogue_update_rogue_reportback_items($updates_to_send_to_rogue);
+
+  if ($response->code != 201) {
+    drupal_set_message($reponse->error, 'warning');
```

Comment:
@sbsmith86 I'm having trouble with this - do we want to display the error on the page if it doesn't go through?

If so, having trouble displaying data to the screen. I've tried the below but nothing seems to be working correctly:
`drupal_set_message(t("%error", array("%error" => $response->error)), "warning");`
`drupal_set_message('<pre>'. var_dump($response->error) .'</pre>', 'warning');`
`drupal_set_message(t('!data', array('!data' => $response->error)), 'warning');`